rpc/v8/subscriptions_test.go

```diff
@@ -1087,11 +1000,24 @@ func assertNextMessage(t *testing.T, conn net.Conn, id SubscriptionID, e *Emitte
 	assert.Equal(t, string(resp), string(got))
 }
 
-func assertNextMessages(t *testing.T, conn net.Conn, id SubscriptionID, emittedEvents []*EmittedEvent) {
+func assertNextTxnStatus(t *testing.T, conn net.Conn, id SubscriptionID, txHash felt.Felt, finality TxnStatus, execution TxnExecutionStatus, failureReason string) {
```

Comment:
Because this is a test it is not immportant, but txHash should be a `*felt.Felt`